src/pkg/packager/deploy.go

```diff
@@ -89,12 +92,18 @@ func (p *Packager) Deploy(ctx context.Context) error {
 		return err
 	}
 	warnings = append(warnings, validateWarnings...)
+	for _, warning := range validateWarnings {
```

Comment:
note: any time I see a simple "take collection, apply function to each" I make this face 😃